src/routes/mainPage/components/Button.module.scss

```
@@ -0,0 +1,59 @@
+@use '/src/styles/constants/colors';
+
+.roundButton {
+  width: 66px;
+  height: 66px;
+  color: colors.$WHITE;
+  background-color: colors.$BUTTON;
+  border-radius: 50%;
+  box-shadow: 0 10px 20px 0 rgba(colors.$BLACK, 15%);
+  transition: 0.3s;
+
+  svg {
+    display: block;
+    margin: auto;
+  }
+
+  &:hover {
+    background-color: rgba(colors.$TEAL, 90%);
+  }
+
+  &:active {
+    background-color: rgba(colors.$INDIGO, 90%);
+  }
+}
+
+.openButton {
+  position: relative;
+  display: block;
+  margin: auto;
+  cursor: pointer;
+}
+
+.addButton {
+  display: hidden;
+  opacity: 0;
+}
+
+.editButton {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: hidden;
+  opacity: 0;
+}
+
+.addButtonOpen {
+  position: absolute;
+  display: inline;
+  opacity: 1;
+  transform: translateY(-70px);
+}
+
+.editButtonOpen {
+  position: absolute;
+  top: 0;
+  display: inline;
+  opacity: 1;
+  transform: translateX(-70px);
+}
```

4개의 속성이 값까지 같으니 위에 @mixin buttonOpen 을 만들어서 가져다가 사용하면 어떨까요??
```suggestion
@mixin buttonOpen {
  position: absolute;
  display: inline;
  opacity: 1;
  transform: translateY(-70px);
}

.addButtonOpen {
  @include buttonOpen;
}

.editButtonOpen {
  @include buttonOpen;
  top: 0;
}
```

---

저도 이게 걸렸는데 mixin 사용하면 되는군요!
transform은 X, Y 따로 줘야해서 수정하고 반영하겠습니다!